android/src/main/java/dog/craftz/sqlite_2/RNSqlite2Module.java

```diff
@@ -110,15 +110,49 @@ public void run() {
   }
 
   // do a update/delete/insert operation
-  private SQLitePLuginResult doUpdateInBackgroundAndPossiblyThrow(String sql, String[] bindArgs,
+  private SQLitePLuginResult doUpdateInBackgroundAndPossiblyThrow(String sql, ReadableArray queryArgs,
                                                                   SQLiteDatabase db) {
     debug("\"run\" query: %s", sql);
     SQLiteStatement statement = null;
     try {
       statement = db.compileStatement(sql);
       debug("compiled statement");
-      if (bindArgs != null) {
-        statement.bindAllArgsAsStrings(bindArgs);
+      // Bind all of the arguments
```

Comment:
Refactoring: It'd be nice to have a separate method for binding all arguments because `doUpdateInBackgroundAndPossiblyThrow` is kind of long now.